packages/api/test/unit/beacon/genericServerTest/debug.test.ts

```diff
@@ -58,5 +58,21 @@ describe("beacon / debug", () => {
       expect(res.wireFormat()).toBe(WireFormat.ssz);
       expect(toHexString(res.ssz())).toBe(toHexString(stateSerialized));
     });
+    it("getHistoricalSummaries", async () => {
```

Comment:
This doesn't really test a lot which would not be already covered by our generic server tests.

To make the generic tests work you need to update the [test data](https://github.com/ChainSafe/lodestar/blob/unstable/packages/api/test/unit/beacon/testData/debug.ts)

Reply:
Switched this over to be part of the generic `testData`

Reply:
Do we need a more comprehensive test that gives a populated response?  I'm not sure the best way to approach this given how your tests are structured but happy to add if it's fruitful and there's an obvious place for it.

Reply:
> Do we need a more comprehensive test that gives a populated response?

The most comprehensive would be to add an e2e test, eg. see https://github.com/ChainSafe/lodestar/blob/64eb0153dd97b7525cb61625b102a6a521869c05/packages/beacon-node/test/e2e/api/impl/beacon/state/endpoint.test.ts#L46

I do think those would be nice-to-have but there is not much internal logic as it's mostly just wiring up existing functions which are tested separately arleady.

Imo for this debug and v0 endpoint we should be fine with the generic tests but if you wanna add an e2e test go for it